### DISCUSSION TITLE
Animagine XL 3.1

### DISCUSSION THREAD

**Discussion by sskit** [lllyasviel/stable-diffusion-webui-forge#1289] [General]
Hello!
During installation, Animagine XL 3.1 model was downloaded with a size of 6.5 GB. I can't find the place where the model is located. The question is, if I uninstall this module his model will be deleted or it will be necessary to delete manually? Where is it located?
And another question. Where are the images generated in these modules saved?

**Comment by lllyasviel** [comment]
all space files can be deleted by space "uninstall" button. 
All remaining files from diffusers or transformers can be deleted by checking the /models/diffusers

**Reply by sskit** [reply]
Thanks! But I don't have that diffusers/transformers folder

I found the model at this address - models\diffusers\models--cagliostrolab--animagine-xl-3.1\snapshots\cde80509086fb3a8e3cdc37d051cd3494edf1ba4

I deleted this module by clicking on the "uninstall" button. But the model file remains. I had to delete it manually.
